signer/src/transaction_coordinator.rs

```diff
@@ -827,6 +857,71 @@ where
         }
     }
 
+    fn authenticate_message(
+        packet: &wsts::net::Packet,
+        public_keys: &hashbrown::HashMap<u32, p256k1::point::Point>,
+        public_key_point: p256k1::point::Point,
+        sender_is_coordinator: bool,
+    ) -> bool {
+        macro_rules! check_signer_public_key {
+            ($message:ident) => {
+                match &public_keys.get(&$message.signer_id) {
+                    Some(signer_public_key) if &public_key_point != *signer_public_key => {
+                        tracing::warn!(
+                            ?packet,
+                            reason = "message was signed by the wrong signer",
+                            "ignoring packet"
+                        );
+                        false
+                    }
+                    None => {
+                        tracing::warn!(
+                            ?packet,
+                            reason = format!("no public key for signer {}", &$message.signer_id),
```

Comment:
I think we should put the `signer_id` as a separate field in the log, so it would more like:
```rust
tracing::warn!(
    ?packet,
    reason = "no public key for signer",
    signer_id = %$message.signer_id,
    ...
```

Reply:
Done